buildspec.yml

```diff
@@ -40,8 +48,12 @@ phases:
         IFS=","
         FULL_VERSIONS=`cat commit_hash`,"latest"
         for VERSION in $FULL_VERSIONS; do
+          # mainn app server image
           docker tag $APP_NAME:$DEPLOYMENT_ENV-`cat commit_hash` $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$APP_NAME:$DEPLOYMENT_ENV-$VERSION
           docker push $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$APP_NAME:$DEPLOYMENT_ENV-$VERSION
+          # ecs task that checks all pages image
+          docker tag $APP_NAME-rake:$DEPLOYMENT_ENV-`cat commit_hash` $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$APP_NAME-rake:$DEPLOYMENT_ENV-$VERSION
+          docker push $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$APP_NAME-rake:$DEPLOYMENT_ENV-$VERSION
         done
 
         IFS=","
```

Comment:
@aaronbrezel do we need to add something for the rake image in the lines below as well? 🤔 I'm guessing this is what forces the `-latest` image to be the, you know, _latest_

Reply:
Actually, we're not going to do this. Following the example of the docs api where we only run put parameter once for the two images that buildspec deploys: https://github.com/WPMedia/newsroom-documents-api#L52